### PR TITLE
Fix queries badly displayed

### PR DESCRIPTION
**Pull request by jaouad-benassila**
connect this PR to ticket #1633
"Is Not" queries are badly displayed in the query section of the cohort builder

### CODE REVIEW

**Comment by nyanofthemoon** [comment]
@jaouad-benassila Will you also update the dependencies in kf-portal-ui?

**Comment by jaouad-benassila** [comment]
Yes I used:
"@kfarranger/admin-ui": "^1.2.2",
"@kfarranger/components": "^1.2.2",
in kf-portal-ui

File: modules/components/src/AdvancedSqonBuilder/sqonPieces/FieldOp.js [hunk 1]
```diff
@@ -56,10 +58,11 @@ export default props => {
                 </span>
                 <span className={`opName`}>{` is ${
                   (Array.isArray(value) && value.length > 1) ||
-                  RANGE_OPS.includes(op)
+                  ((RANGE_OPS.includes(op) || TERM_OPS.includes(op)) && !(isEqual(value,["true"]) || isEqual(value,["false"])))
```
Comment (nyanofthemoon) [inline modules/components/src/AdvancedSqonBuilder/sqonPieces/FieldOp.js:61]: @jaouad-benassila This probably makes sense, but I don't understand? `!(isEqual(value,["true"]) || isEqual(value,["false"])`

**Comment by nyanofthemoon** [comment]
@jaouad-benassila I mean once we approve this PR, we need to push these changes to NPM, which means that `@kfarranger/arranger` will change from version `1.2.2` to `1.2.3` in the forementioned repositories. I will help you out tomorrow :)

File: modules/components/src/AdvancedSqonBuilder/sqonPieces/FieldOp.js (hunk 1 above)
Comment (jaouad-benassila) [reply modules/components/src/AdvancedSqonBuilder/sqonPieces/FieldOp.js:61]: TERM_OPS is used for TermFilter
RANGE_OPS is used for RangeFilter
!(isEqual(value,["true"]) || isEqual(value,["false"]) is used to exclude boolean filter

**Comment by jaouad-benassila** [comment]
@nyanofthemoon   You mean to test the fix ?

**Review by nyanofthemoon — APPROVED** [review]

**Pull request merged**